### DISCUSSION TITLE
The Changes of Views Folders' Structure in v2

### DISCUSSION THREAD

**Discussion by cubemastercodes** [rappasoft/laravel-livewire-tables#750] [General]
Hi @rappasoft, 

I really love this package as I use it as a base for any project I started. 

Thank you a lot for releasing v2 with big enhancements: setting configurations, filters, configurable areas, mobile collapse... and more. 

My concern is: why the changes of old views structure i.e.

v1: 
just located in the required theme folder

```
tailwind
bootstrap-4
bootstrap-5
```

v2
checking on almost every view for the theme.

```
@php
    $theme = $component->getTheme();
@endphp

@if ($theme === 'tailwind')
```

It is really memory-consuming to check the theme in every file not to mention long file content.

I really think the structure of v1 views will be better, what do you think?

**Comment by rappasoft** [comment]
I haven't had any memory issues. I did it this way so I can see all the code for a specific component in one file and have 1/3 the number of files to manage.

**Reply by cubemastercodes** [reply]
Thank you a lot... appreciated your work 🌷
